### ISSUE TITLE
Regarding making liveries.

### ISSUE THREAD

**Issue by airgeorge** [kolos26/GEOFS-LiverySelector#101]
Many of you may have noticed i have not been making as much liveries recently. This is mainly because of school,it turns out that it can be really hard to make a bunch of liveries and balance school on top of that. Because of that i will be taking a little break until December 1st. I want you to know that this was not an easy decision giving up something that i love doing so much,but I have to do it to keep up with my grades.But before I go i have one more livery,China Southern A380! (also can you add a screen recording so i can see how a livery is added,thank you!) I cant wait to see how much this community will grow when i get back! 

Best Regards
           -George
![image](https://user-images.githubusercontent.com/112053353/197270096-2d0e9556-0d55-4aff-a3fd-e78b490e4a95.png)

**Comment by Ryubunao** [comment]
Last words from a legend..

**Comment by BoeingBus747** [comment]
I can't wait for you to be back!

**Comment by Egan-31** [comment]
NOOOOOOOOOOOOOOO ! I was working on it for 2 weeks :( Not important, i'll be faster next time. more seriously, i think i shoul prevent when i start a long time project.

**Comment by kolos26** [comment]
@airgeorge 
Thank you for supporting LiverySelecor with so many beautiful liveries! I totally understand your problem and hope you will get good marks. Hope to see you soon, winter holiday is coming.

Best regards,
Kolos

**Comment by airgeorge** [comment]
@kolos26 one last livery ;). now
 Best Regards,
        -George
![image](https://user-images.githubusercontent.com/112053353/197898191-48c804cd-fcb6-4730-8718-4fe281dd4df5.png)

**Comment by BoeingBus747** [comment]
> @kolos26 one last livery ;). now Best Regards, -George ![image](https://user-images.githubusercontent.com/112053353/197898191-48c804cd-fcb6-4730-8718-4fe281dd4df5.png)

NO, I was making that, well I'm gonna make the etihad year of zayed 2018, no one steal it, okay?

**Comment by BlueWacki** [comment]
is cool

**Comment by kolos26** [comment]
@airgeorge Thanks for this (temporary last) livery.
@BoeingBus747 The Etihad one would be cool. If you create that you can also create the general Etihad livery 'cause they're almost the same.

**Comment by BoeingBus747** [comment]
> @airgeorge Thanks for this (temporary last) livery. @BoeingBus747 The Etihad one would be cool. If you create that you can also create the general Etihad livery 'cause they're almost the same.

It's not the weekend so I can't make much progress rn so I should be done in Friday/Saturday and I'll also do the original one at the same time

**Comment by kolos26** [comment]
> > @airgeorge Thanks for this (temporary last) livery. @BoeingBus747 The Etihad one would be cool. If you create that you can also create the general Etihad livery 'cause they're almost the same.
> 
> It's not the weekend so I can't make much progress rn so I should be done in Friday/Saturday and I'll also do the original one at the same time

This is not a horse-race ;) Just do it if you have time and mood.

**Comment by BoeingBus747** [comment]
> > > @airgeorge Thanks for this (temporary last) livery. @BoeingBus747 The Etihad one would be cool. If you create that you can also create the general Etihad livery 'cause they're almost the same.
> > 
> > 
> > It's not the weekend so I can't make much progress rn so I should be done in Friday/Saturday and I'll also do the original one at the same time
> 
> This is not a horse-race ;) Just do it if you have time and mood.

Yeah I do, plus I can literally save the part without text and then do both of them

**Comment by BoeingBus747** [comment]
> Reference in new issue

Great! :D Anyhow, I'm probably going to need a break since biology is getting tough this year

**Comment by BlueWacki** [comment]
is close to 12/1/22

**Comment by BoeingBus747** [comment]
I hate it when people use the Month/Day/Year format

**Comment by airgeorge** [comment]
the only reason i use it is because whenever the school makes us write dates we have to do it in that format
